PowerPointLabs/PowerPointLabs/RemoveHighlighting.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.Office.Interop.PowerPoint;
+using PowerPointLabs.Models;
+
+namespace PowerPointLabs
```

Comment:
can put this new class into its feature's package?
maybe highlightlabs or something